Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/SimplifiedLogin/SimplifiedLoginManagerTests.swift

```diff
@@ -50,6 +50,141 @@ final class SimplifiedLoginManagerTests: XCTestCase {
+        let userProfileResponse = Fixtures.userProfileResponse
+        let mockHTTPClient = MockHTTPClient()
+        stub(mockHTTPClient) { mock in
+            when(mock.execute(request: any(), withRetryPolicy: any(), completion: anyClosure()))
+                .then { (_, _, completion: HTTPResultHandler<UserContextFromTokenResponse>) in
+                    completion(.success(contextResponse))
+                }
+            when(mock.execute(request: any(), withRetryPolicy: any(), completion: anyClosure()))
+                .then { (_, _, completion) in
+                    completion(.success(SchibstedAccountAPIResponse(data: userProfileResponse)))
+                }
+        }
+        
+        // MockClient
+        let jwks = RemoteJWKS(jwksURI: Fixtures.clientConfig.serverURL.appendingPathComponent("/oauth/jwks"), httpClient: mockHTTPClient)
+        let tokenHandler = TokenHandler(configuration: Fixtures.clientConfig, httpClient: mockHTTPClient, jwks: jwks)
+        let mockClient = MockClient(configuration: Fixtures.clientConfig,
+                                      sessionStorage: MockSessionStorage(),
+                                      stateStorage: StateStorage(storage: MockStorage()),
+                                      httpClient: mockHTTPClient,
+                                      jwks: jwks,
+                                      tokenHandler: tokenHandler)
+        let returnedUserSession = UserSession(clientId: "aClientId", userTokens: Fixtures.userTokens, updatedAt: Date())
+        mockClient.userSessionToReturn = returnedUserSession
+        
+        let expectation = self.expectation(description: "Should fail with SimplifiedLoginError.noVisibleViewControllerFound")
+        let sut = SimplifiedLoginManager(client: mockClient, contextProvider: ASWebAuthSessionContextProvider(), env: .pre, completion: { result in })
+        sut.requestSimplifiedLogin("A name", completion: { result in
+            switch result {
+            case .failure(SimplifiedLoginManager.SimplifiedLoginError.noVisibleViewControllerFound):
+                expectation.fulfill()
+            default:
+                XCTFail("Should fail with SimplifiedLoginError.noVisibleViewControllerFound")
+            }
+        })
+        
+        wait(for: [expectation], timeout: 0.2)
+    }
+    
+    func testRequestSimplifiedLogin_failingProfileResponse() {
+        // MockHTTPClient
+        let userProfileResponse = Fixtures.userProfileResponse
+        let mockHTTPClient = MockHTTPClient()
+        let expectedError: Result<UserContextFromTokenResponse, HTTPError> = .failure(HTTPError.unexpectedError(underlying: LoginStateError.notLoggedIn))
+        stub(mockHTTPClient) { mock in
+            when(mock.execute(request: any(), withRetryPolicy: any(), completion: anyClosure()))
+                .then { (_, _, completion: HTTPResultHandler<UserContextFromTokenResponse>) in
+                    completion(expectedError)
```

Comment:
Is it necessary to return this error from mockHTTPClient. 
I see that we return this error when uuid is not present in User. We dont do network request. For reference: https://github.com/schibsted/account-sdk-ios-web/blob/a229379163ae92a9ced5acf3bea61f90e8eb201c/Sources/AccountSDKIOSWeb/Lib/API/SchibstedAccountAPI.swift#L164